src/components/Snackbar/useSnackbar.tsx

```diff
@@ -39,7 +39,7 @@ export function useSnackbar(): UseSnackbarHook {
   }, []);
 
   const triggerNotice = useCallback(
-    (props) => {
+    (props: any) => {
```

Comment:
**suggestion:** I think we could change this `any` to `TriggerNoticeProps`. 
```suggestion
    (props: TriggerNoticeProps) => {
```

Reply:
Sweet, changed!